src/main/java/org/fcrepo/spec/testsuite/versioning/LdprmHttpOptions.java

```diff
@@ -123,16 +152,22 @@ public void ldprmMustNotSupportPut(final String uri) {
                                         ps);
 
         //create an LDPRm
+        final String mementoURI = createMemento(uri, info);
+        final Response mementoResponse = doGet(mementoURI);
         //verify that a subsequent OPTIONS does not return PUT
+        confirmAbsenceOfHeaderValueInMultiValueHeader("Allow", "PUT", mementoResponse);
         //verify that issuing a PUT returns a 405 (method not allowed).
+        final Response response = doPutUnverified(mementoURI);
+        assertEquals(response.getStatusCode(), 405, "Response to a PUT should be 405");
```

Comment:
same comment as above about the specific 405 error - should probably just check for a 4xx error.